README.md

```diff
@@ -1 +1,33 @@
-"This document will highlight the bets and worst practicies realting to developing software in a team" 
+### What Is Software?
+_____
+ 
+Software is defined as the programs and other operation information used by a computer.Examples of this are the operating systems that your computer uses or the applications that are on your phone The development of software has changed over the years as technology has advanced, it has gone from one person or a small group of individuals working on one project to large companies with developers all over the world contributing to one uniform goal.One of the main problems that has stayed present even up until today is developing software as a team
+ 
+### Programming Together
+_____
+Working in teams has always been deemed the fastest way to get work done in any industry, the same applies to software development.
+ 
+Many benefits have been associated with developing in a team environment for example:
+ 
+* **Fewer Coding Mistakes**
+Because there is more than one programmer looking over your work, it results in better code.
+ 
+* **Sharing Knowledge**
+Developers working together share knowledge amongst themselves which is better than accessing online tutorials etc. Developers can also pick up best practices and better techniques from more advanced developers.
+ 
+* **Develops your staff’s interpersonal skills**
+Collaborating on a single project helps your team to appreciate the value of communication and teamwork.
+ 
+However with these benefits also comes problems, as expected one must determine how these teams communicate with each other in an effective manner which will prevent things such as unpredictable progress and defects in the application being developed
+ 
+### Software Quality Handbook
+____
+This document will highlight the best and worst practices relating to developing software in a team
+ 
+**Topics Covered:**
+* [Task Estimation](../SQA/TaskEstimation/TaskEstimation.md)
+* [Coding Standards](../SQA/coding-standards/CodingStandards.md)
+* [Code Reviews](../SQA/CodeReviews/CodeReviews.md)
```

Comment:
Nice! You used relative paths here :)
```suggestion
* [Task Estimation](../TaskEstimation/TaskEstimation.md)
* [Coding Standards](../coding-standards/CodingStandards.md)
* [Code Reviews](../CodeReviews/CodeReviews.md)
```
I think "SQA" can be removed from the path, as it doesn't make a difference

Reply:
They actually don't work for me this way probably cause of the way i have arranged my project directory if your able to get this working in the way you show please go for it and make the change :+1:

Reply:
```suggestion
* [Task Estimation](askEstimation/TaskEstimation.md)
* [Coding Standards](coding-standards/CodingStandards.md)
* [Code Reviews](CodeReviews/CodeReviews.md)
```
This should work. Forgot to mention to remove "../" as well